### ISSUE TITLE
Have you tried to use a bigger text encoder?

### ISSUE THREAD

**Issue by Rainlt** [GeWu-Lab/MUSIC-AVQA#5]
I have tried to use a pretrained BERT to encode the text, but no matter how I tune the hyperparameters, the accuracy is around 50%. It seems that these text shouldn't be encoded with a complex encoder? Could you please tell me your opinion on this phenomenon?

**Comment by ayameyao** [comment]
Thank you very much for your interest in our work.

We believe that a thorough understanding of the input question is crucial in AVQA tasks. This allows the model to focus better on video and audio content relevant to the question, resulting in improved performance when answering questions.

Furthermore, we have experimented with more powerful models such as CLIP for text encoding, which outperformed the LSTM-based approach. We suggest trying other versions of text encoders or using the Baseline method provided in the code to quickly verify the results obtained after processing with different text encoders.

If you have any further questions, please do not hesitate to contact us at guangyaoli@ruc.edu.cn.

**Comment by Rainlt** [comment]
> Thank you very much for your interest in our work.
> 
> We believe that a thorough understanding of the input question is crucial in AVQA tasks. This allows the model to focus better on video and audio content relevant to the question, resulting in improved performance when answering questions.
> 
> Furthermore, we have experimented with more powerful models such as CLIP for text encoding, which outperformed the LSTM-based approach. We suggest trying other versions of text encoders or using the Baseline method provided in the code to quickly verify the results obtained after processing with different text encoders.
> 
> If you have any further questions, please do not hesitate to contact us at [guangyaoli@ruc.edu.cn](mailto:guangyaoli@ruc.edu.cn).

Thanks a lot!
